app/src/main/java/com/andrewKarachun0304/tmshomeworkandroid/MainActivity.kt

```diff
@@ -3,16 +3,21 @@ package com.andrewKarachun0304.tmshomeworkandroid
 import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
 import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+    }
```

Comment:
The same case as in comment 1

Reply:
![image](https://user-images.githubusercontent.com/60601044/94237563-bc7f2000-ff17-11ea-9f14-68e58766cc73.png)
He is making it from xml